src/status_im/ui/screens/events.cljs

```diff
@@ -228,9 +243,9 @@
   :initialize-account-db
   (fn [{:keys [accounts/accounts networks/networks network view-id navigation-stack
                status-module-initialized? status-node-started?]
-        :or [network (get app-db :network)]
-        :as db} [_ address]]
-    (-> app-db
+        :or [network (get db :network)]
+        :as db} [_ address]] 
+    (-> db
```

Comment:
@rasom yeah it's because of this I think he fixed it already

Reply:
Yes, it's actually broken in develop now...

Reply:
i was using this commit `0d1314dd59d95de2430d2d7ea1f093204e78e4be`

Reply:
can't reproduce this in `develop`

Reply:
It was a misunderstanding, the thing is that this event now starts with a fresh db so you need to explicitely pass fields if you want them to be passed between accounts.

Reply:
So ok, just to clarify, should it be fixed in `develop` or in this PR? Because i can't reproduce this bug in develop

Reply:
This PR should be fixed